modules/ROOT/pages/salesforce/salesforce-connector-97.adoc

```diff
@@ -0,0 +1,1274 @@
+NOTE: A channel or topic must be published to Salesforce before a subscription to the channel can be created.
++
+* *Replay Topic* - Subscribe to a topic. This trigger provides topic-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place on records in the specified topic trigger the flow.
+* *Subscribe Channel* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events that take place after you subscribe.
+* *Subscribe Topic* - Subscribe to a topic. This trigger provides topic-related notification settings for new events that take place after you subscribe.
+
+
+== Configure a Salesforce New Object Trigger
+
+The following shows how to configure the Salesforce New Object trigger.
+
+. Create a New Mule Application in Design Center.
+. Set the trigger. For example, you can use Salesforce > New Object.
+. Click *Set up*.
+. Specify the Connection Type:
++
+[%header%autowidth.spread]
+|===
+|Connection Type |Salesforce Information Site
+| <<Configure Basic Username Password Authorization, Basic Username Password>> |https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_login.htm[SOAP API Username Password Login]
+| <<Configure OAuth 2.0 Authorization, OAuth 2.0>> |https://help.salesforce.com/articleView?id=remoteaccess_oauth_web_server_flow.htm[OAuth 2.0 Web Server Authorization Flow]
+| <<Configure OAuth JWT Authorization, OAuth JWT>> |https://help.salesforce.com/articleView?id=remoteaccess_oauth_jwt_flow.htm[OAuth 2.0 JWT Bearer Token Flow]
+| <<Configure OAuth SAML Authorization, OAuth SAML>> |https://help.salesforce.com/apex/HTViewHelpDoc?id=remoteaccess_oauth_SAML_bearer_flow.htm[OAuth 2.0 SAML Bearer Assertion Flow]
+| <<Configure OAuth Username Password Authorization, OAuth Username Password>> |https://help.salesforce.com/articleView?id=remoteaccess_oauth_username_password_flow.htm[OAuth 2.0 Username-Password Flow]
+|===
++
+. Specify the parameters as required for each configuration type, and click *Test* to ensure you have a valid connection to Salesforce.
+. If needed, configure the *Apex* tab to decide what metadata to fetch and to add Apex class names.
+. If you haven't specified a `Connection Type`, and if needed, configure the *Advanced* tab to set the `Max Idle Time` and `Time Units` for how long to hold open a connection before it expires.
+. After you specify a connection type, if needed, you can use the *Advanced* tab to change values for the Reconnection Strategy, Pooling Profile, and Expiration Policy.
+. If needed, set the `Redelivery Policy` on the main configuration screen. This indicates the maximum number of tries to deliver trigger content.
+
+=== Configure Salesforce as a Component
+
+After creating a trigger, click the plus sign. If needed, click *Transform* to use DataWeave to map fields between your received data and what to use with your Salesforce component.

```

Comment:
"what to use with your Salesforce component" - is there another way to say this?